tests/visualization_tests/matplotlib_tests/test_contour.py

```diff
@@ -105,11 +111,15 @@ def fail_objective(_: Trial) -> float:
         plot_contour(study, ["optuna", "Optuna"])
 
     figure = plot_contour(study, params=params)
+    plt.savefig(BytesIO())
```

Comment:
Could you move this to the end of asserts for this figure? I suppose around L131 in the current version.
I refer to this type of comment as [to end of the test case] in the following RP comment.